README.md

```diff
@@ -1,74 +1,105 @@
 
-## Description
+# Description
+
+> This is grpc-gateway project for working with theaters via database theater.
+>
+>grpc-server runs on 8181 port. gateway runs on 8080 port. database runs on 5432 port.
+>
+>To work with products use this address: ```localhost:8181/{path}```
+>
+>To run this application locally you need to clone this project and also you need to have active docker server.
 
 ##### Backend Application for storing theater
 
 |Path|Method|Description|
 |:---:|:---:|:---:|
-|```/tickets```|```GET```|get all tickets|
-|```/posters```|```GET```|get all posters|
-|```/users?idAccount={id}```|```GET```|get all users by ```account```|
-|```/account?id={id}```|```GET```|get account by ```id```|
-|```/genre?id={id}```|```GET```|get genre by ```id```|
-|```/hall?id={id}```|```GET```|get hall by ```id```|
-|```/location?id={id}```|```GET```|get location by ```id```|
-|```/performance?id={id}```|```GET```|get performance by ```id```|
-|```/place?id={id}```|```GET```|get place by ```id```|
-|```/poster?id={id}```|```GET```|get poster by ```id```|
-|```/price?id={id}```|```GET```|get price by ```id```|
-|```/role?id={id}```|```GET```|get role by ```id```|
-|```/schedule?id={id}```|```GET```|get schedule by ```id```|
-|```/sector?id={id}```|```GET```|get sector by ```id```|
-|```/ticket?id={id}```|```GET```|get ticket by ```id```|
-|```/user?id={id}```|```GET```|get user by ```id```|
-|```/account?id={id}```|```DELETE```|delete account by ```id```|
-|```/genre?id={id}```|```DELETE```|delete genre by ```id```|
-|```/hall?id={id}```|```DELETE```|delete hall by ```id```|
-|```/location?id={id}```|```DELETE```|delete location by ```id```|
-|```/performance?id={id}```|```DELETE```|delete performance by ```id```|
-|```/place?id={id}```|```DELETE```|delete place by ```id```|
-|```/poster?id={id}```|```DELETE```|delete poster by ```id```|
-|```/price?id={id}```|```DELETE```|delete price by ```id```|
-|```/role?id={id}```|```DELETE```|delete role by ```id```|
-|```/schedule?id={id}```|```DELETE```|delete schedule by ```id```|
-|```/sector?id={id}```|```DELETE```|delete sector by ```id```|
-|```/ticket?id={id}```|```DELETE```|delete ticket by ```id```|
-|```/user?id={id}```|```DELETE```|delete user by ```id```|
+|```/api/v1/tickets```|```GET```|get all tickets|
+|```/api/v1/posters```|```GET```|get all posters|
+|```/api/v1/users/{id}```|```GET```|get all users by ```account id```|
+|```/api/v1/account/{id}```|```GET```|get account by ```id```|
+|```/api/v1/genre/{id}```|```GET```|get genre by ```id```|
+|```/api/v1/hall/{id}```|```GET```|get hall by ```id```|
+|```/api/v1/location/{id}```|```GET```|get location by ```id```|
+|```/api/v1/performance/{id}```|```GET```|get performance by ```id```|
+|```/api/v1/place/{id}```|```GET```|get place by ```id```|
+|```/api/v1/poster/{id}```|```GET```|get poster by ```id```|
+|```/api/v1/price/{id}```|```GET```|get price by ```id```|
+|```/api/v1/role/{id}```|```GET```|get role by ```id```|
```

Comment:
create separate table for each entity, this allows user read your readme file easier